src/Nuget.Dockerfile

```diff
@@ -0,0 +1,46 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+FROM mcr.microsoft.com/dotnet/aspnet:8.0.3-alpine3.19-amd64@sha256:a531d9d123928514405b9da9ff28a3aa81bd6f7d7d8cfb6207b66c007e7b3075 AS runtimebase
+
+LABEL maintainer="Ed-Fi Alliance, LLC and Contributors <techsupport@ed-fi.org>"
+
+RUN apk --no-cache add bash=~5 gettext=~0 postgresql16-client=~16
+
+FROM runtimebase AS setup
+
+ENV LOG_LEVEL=${LOG_LEVEL}
+#replace with latest
+ARG VERSION=0.0.0-alpha.0.113
```

Comment:
@stephenfuqua I think the answer is yes, but I would like to confirm if the Version value should be latest

Reply:
`latest` is not a valid tag for a NuGet package.

Reply:
Is it possible to have an `ARG` without a default value? I can't remember. If so, might as well remove the default value above - since it will no longer be correct as soon as the next pull request merges.

Reply:
As discussed in daily standup, we can leave this in place.